app/controllers/admin/organisations_controller.rb

```diff
@@ -94,9 +102,14 @@ def destroy
 
 private
 
+  def organisation_roles(type)
+    @organisation.organisation_roles.joins(:role)
+                 .merge(Role.send(type)).order(:ordering)
```

Comment:
we shouldn't really be using `send` as it doesn't respect an objects interface i.e. it will work with private methods.
https://stackoverflow.com/questions/30401970/what-is-the-difference-between-rubys-send-and-public-send-methods

we can use `public_send` instead

```
.merge(Role.public_send(type)).order(:ordering)
```

Reply:
Cool, thank you. I hadn't appreciated that. Good to know!